src/main/python/interpolate.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+'''
+Anserini: A Lucene toolkit for replicable information retrieval research
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import argparse
+import collections
+import pytrec_eval
+
+
+def load_qrels(path):
+    qrels = collections.defaultdict(dict)
+    for line in open(path):
+        line = ' '.join(line.split())
+        query_id, _, doc_id, relevance = line.split(' ')
+        qrels[query_id][doc_id] = int(relevance)
+    return qrels
+
+
+def load_run(path):
+    run = collections.defaultdict(dict)
+    for line in open(path):
+        query_id, _, doc_id, _, score, _ = line.split(' ')
+        run[query_id][doc_id] = float(score)
+
+    return run
+
+
+parser = argparse.ArgumentParser(
+    description='Interpolate 2 runs by first normalizing their scores (between'
+                '0 and 1) and then doing: alpha * run1_score + (1 - alpha) * '
+                'run2_score.')
```

Comment:
Are you setting width=100? These lines seem really short to need linebreaks...

Reply:
I'm using PEP8, which is 80 chars

Reply:
> Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters (effectively increasing the maximum length to 99 characters), provided that comments and docstrings are still wrapped at 72 characters.

https://pep8.org/#maximum-line-length

haha, can we agree? ;)

Reply:
haha, ok, I do prefer longer lines